apis/v1beta1/types.go

```diff
@@ -167,6 +168,10 @@ type VirtualMachineCloneSpec struct {
 	// virtual machine is cloned.
 	// +optional
 	NumCoresPerSocket int32 `json:"numCoresPerSocket,omitempty"`
+	// resources is the definition of the VM's cpu and memory
+	// reservations, limits and shares.
+	// +optional
+	Resources VirtualMachineResources `json:"resources,omitempty,omitzero"`
```

Comment:
```suggestion

	// resources is the definition of the VM's cpu and memory
	// reservations, limits and shares.
	// +optional
	Resources VirtualMachineResources `json:"resources,omitempty,omitzero"`

```

Reply:
If I'm not mistaked, this struct is shared between govmomi mode and supervisor mode.
If this is correct, impacts of this changes must me considered for both modes.